opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -746,13 +747,18 @@ struct Mover<DataHandle,3> : public BaseMover<DataHandle>
 
 template<class DataHandle>
 void CpGridData::scatterData(DataHandle& data, CpGridData* global_data,
-                          CpGridData* distributed_data)
+                             CpGridData* distributed_data, const InterfaceMap& inf)
 {
 #if HAVE_MPI
     if(data.contains(3,0))
-       scatterCodimData<0>(data, global_data, distributed_data);
+    {
+        Entity2IndexDataHandle<DataHandle, 0> data_wrapper(*global_data, *distributed_data, data);
+        communicateCodim<0>(data_wrapper, ForwardCommunication, inf);
```

Comment:
This seems to be the focus of the change, what is the advantage to using communicateCodim() rather than scatterCodimData()? Both are using MPI at a lower level I assume? Also, for vertex data scatterCodimData() is still used, do you intend to replace that also with communicateCodim()?